swan-lake/learn/observing-ballerina-code.md

```diff
@@ -65,70 +67,61 @@ service /hello on new http:Listener(9090) {
 
 ### Step 4 - Observing the 'Hello World' Ballerina Service
 
-Observability is disabled by default and can be enabled by using the `--b7a.observability.enabled=true` flag or updating the configurations.
+By default, Observability is not included in the executable created by Ballerina. It can be added
```

Comment:
```suggestion
By default, observability is not included in the executable created by Ballerina. It can be added
```

Reply:
Fixed in 146f6434f70aee59d33f47a274e40ed2ac703d6a